book/aave_proposals.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
+ "cells": [
```

Comment:
My general comment on this report, cc @vaughnmck for further review (@vaughnmck you can look at the rendered Jupyter notebook by clicking on View File 
<img width="673" alt="image" src="https://user-images.githubusercontent.com/1884912/233058640-bb08a190-cf41-456e-a3e3-df8a894fa0e7.png">
).

> We take a look at 5 recent proposals on Aave Governance, including the first Butter Aave Elections, and compare the voter turnout and the "pivotality" (a measure of whale influence) of each of them.

This is not what we planned. We want to take a look at the top 10 Aave proposals on Snapshot ranked by Voting Power and #Voter. Here there are, based on @vaughnmck 's analysis:

<google-sheets-html-origin><style type="text/css"><!--td {border: 1px solid #cccccc;}br {mso-data-placement:same-cell;}--></style>

Title | Total VP | Voters | Total VP Rank | Voter Rank | Total Rank | ID
-- | -- | -- | -- | -- | -- | --
[TEMP CHECK] Incentivized Delegate Campaign (3-month) | 742,542.48 | 17,658.00 | 6.00 | 3.00 | 9.00 | 0xf6da417983653f7e7f5560a8e16ad77d5e25533bba9157abfd222347b40d09d0
[TEMP CHECK] - Launch Aave V3 on Starknet | 665,006.49 | 21,321.00 | 16.00 | 1.00 | 17.00 | 0xde90f4d29c4c5dd55cba023b29bdde8c49e37f70c3398668a78e2f173a61b4e6
Safety Module upgrade to v1.5 | 728,994.69 | 16,152.00 | 7.00 | 20.00 | 27.00 | 0x743ae6001bb0fdfa9a8c4475a2489c20917a79084725f0101ac7c1e7f0cf27c6
Deploy Aave V3 to Metis Andromeda Mainnet | 717,974.83 | 16,126.00 | 9.00 | 22.00 | 31.00 | 0x7e6175c21342dbd88dc5b0dabf2d0a86339e3ef3f199fc7d778a9a6f05ed0df9
[ARFC] Add MKR to Ethereum v3 | 660,111.69 | 16,354.00 | 17.00 | 17.00 | 34.00 | 0xf4aec3fbab5096752be96f0e5b522f37318c1902cf8b897b049b7a94d478de73
[TEMP CHECK] Aave V3 deployment on BNB Chain | 586,055.57 | 16,976.00 | 32.00 | 6.00 | 38.00 | 0x4579b562a4ea07c52871f724a8baf156514a97384e9427156d04fb6f0ae91515
[ARFC] Grant OP Emission_Admin for Optimism v3 Liquidity Pool to Lido DAO | 579,100.80 | 17,663.00 | 38.00 | 2.00 | 40.00 | 0xd8a400fc731b6e807f4d89aa1fa47225f0dfaaa8fc49ef04b46b11b98fbfa15f
[ARFC] Add UNI to Ethereum v3 | 618,327.88 | 16,482.00 | 28.00 | 12.00 | 40.00 | 0x51d67ef69e901b34f1d111f2cd5d582c59cffa8d70b7939023febd20f7613b88
[ARFC] Deploy aCRV & CRV to veCRV | 585,810.01 | 16,462.00 | 33.00 | 13.00 | 46.00 | 0xf92c5647c7f60a4a3db994b4953fc4408f5946cafdc0cebcd4c5924f40e04d36
[ARFC] Add SNX to Ethereum v3 | 619,021.56 | 16,240.00 | 27.00 | 19.00 | 46.00 | 0x5f232a89e10d67df3aad2907e8dce3bec9708596929b3254055cf37499969b89

Can you please update the analysis with this?

## Below my wording comments

> We take a look at 5 recent proposals on Aave Governance, including the first Butter Aave Elections, and compare the voter turnout and the "pivotality" (a measure of whale influence) of each of them.

replace with

> We take a look at the top 10 Aave Snapshot proposals ranked by voting power and voters count, includes the first Butter Aave Elections.
> 
> We compare their voter turnout and _pivotality_ metric (a measure of whale influence, described in our previous [Mirror article](https://mirror.xyz/butterd.eth/_VrHsBXZU0IYuIUMq0ifJTffvH0e1AKNh6I-BIV3Nc4) and [data report]()).

Reply:
Thanks for your review!